src/msw/webview_ie.cpp

```diff
@@ -963,14 +964,14 @@ bool wxWebViewIE::IsElementVisible(wxCOMPtr<IHTMLElement> elm)
             if(SUCCEEDED(elm2->get_currentStyle(&style)))
             {
                 //Check if the object has the style display:none.
-                if((style->get_display(&tmp_bstr) != S_OK) || 
-                   (tmp_bstr != NULL && (wxCRT_StricmpW(tmp_bstr, L"none") == 0)))
+                if((style->get_display(display_bstr) != S_OK) || 
+                   ((BSTR)display_bstr != NULL && (wxCRT_StricmpW(display_bstr, L"none") == 0)))
```

Comment:
The reason I did not add such conversion was the same why I did not add it for `wxCOMPtr`. Additionally, `BSTR` is kind of an oddity among pointers, as NULL `BSTR` is still a valid (empty) string.

I was actually tempted to rewrite the expression above but I did not want to make any unnecessasry changes. The expression should be changed from
```cpp
((BSTR)display_bstr != NULL && (wxCRT_StricmpW(display_bstr, L"none") == 0))
```
to
```cpp
wxString(display_bstr).IsSameAs(wxS("none"), false)
```
and similarly for the other expression.